specification/PropertiesReference_3dtiles.adoc

```diff
@@ -0,0 +1,3262 @@
+
```

Comment:
Could the property references be grouped together in a more consistent way, like tileset properties first, then tile format properties, then metadata - roughly based on the folder structure on github? I think it will be easier to follow that way.

![property-reference-ordering](https://user-images.githubusercontent.com/915398/170698385-60c2d426-cf36-4604-add5-e3a8b5fb4528.png)

Reply:
Right now, this is listed alphabetically by the "type name", e.g.
```
...
  templateUri (linked)
  tile (linked)
  tile.implicitTiling (linked)
  tileset (linked)
...
```
Of course, abritrary other orders can be established (e.g. sorting by the "Title" - i.e. the string that is actually displayed at the end). But when it comes to "grouping" and "folders", I'll have to see how the "desired sorting" can be squeezed into wetzel - beyond the `this.knownTypeNames = Object.keys(this.knownTypes).sort();` that it is doing now...